utils/AxiosRequest.js

```diff
@@ -1,14 +1,16 @@
 const axios = require('axios').default;
 
-const localUrl = '192.168.68.102';
+const localUrl = '172.20.10.4';
 const apiUrl = `http://${localUrl}:8000`;
 
-export const fetchPredictions = async (userId, noOfRecommendations) => {
+export const fetchPredictions = async (userId, startIndex, noOfRecommendations) => {
```

Comment:
Shouldn't there also be a backend PR too with updated request model?

Reply:
jeps, den ligger i preds branchen. my bad haha